pkg/actors/actors.go

```diff
@@ -138,6 +138,8 @@ const (
 	incompatibleStateStore = "state store does not support transactions which actors require to save state - please see https://docs.dapr.io/operations/components/setup-state-store/supported-state-stores/"
 )
 
+var ErrInvokeDotnetActor = errors.New("error invoking .NET actor")
```

Comment:
Can your remove all occurrences specific to .NET? Just because the header itself might be used either now or in the future by other SDKs (perhaps ones that are not in the `dapr` org, for example).